### DISCUSSION TITLE
The Repository is Excluded from Hactoberfest

### DISCUSSION THREAD

**Discussion by SidharthMudgil** [Midway91/HactoberFest2023#175] [General]
This repository don't follow the hactoberfest guidelines so it is been marked as excluded means no pr merge will be counted in hactoberfest.

**Comment by SidharthMudgil** [comment]
What should I do then? If you are still a beginner or it's your first time in OpenSource this is a good start to learn how to create a pull request.

**Comment by Midway91** [comment]
@SidharthMudgil I don't know why it says that I have the hacktoberfest tag in the topics area of the repository and I add hacktoberfest-accepted tag to every pull request.

**Comment by SidharthMudgil** [comment]
I can't understand what you're asking... Like this repository is showing under hactoberfest topic? That what you're asking?

**Reply by Midway91** [reply]
@SidharthMudgil What I am trying to say it that I have done everything that the hacktoberfest website says to do.

**Reply by Midway91** [reply]
@SidharthMudgil can you make a simple test pull request titled something like hacktoberfest verification test. Because I want to try something new to see if it works. Thank you

**Reply by SidharthMudgil** [reply]
Once marked as excluded it can't be changed... I'll suggest you to drop your personal project like a website or Android app... anything and then maintain that project and only accept good prs that actually are beneficial to your project.

**Comment by SidharthMudgil** [comment]
Yes, its because the contribution in this repository are mot good quality... you're accepting all pull requests that will be labelled as SPAM, why? Because it's easy way of completing all 4 prs required for hactoberfest. That way Contributor will not learn anything...

But don't worry it doesn't effect your GitHub pr anything it's just hactoberfest... alsp it's still a good repo for contributors who are news and want to create their first pr.

**Reply by Midway91** [reply]
@SidharthMudgil Ok thanks for the info.
